cjs/index.js

```diff
@@ -20,11 +20,15 @@ try {
     });
     define(proto, 'dispatchEvent', function (event) {
       var secret = wm.get(this);
-      var listeners = secret[event.type];
-      if (listeners) {
+      if (secret[event.type]) {
+        var listeners = secret[event.type].slice(0);
         define(event, 'target', this);
         define(event, 'currentTarget', this);
-        listeners.slice(0).forEach(dispatch, event);
+        
+        // if an event listener invoke stopImmediatePropagation it should stop calling other listeners
+        for (var i = 0; i < listeners.length && !event._stopImmediatePropagationFlag; i++) {
+          dispatch.call(event, listeners[i]);
```

Comment:
so here we have to get the flag from the Event object and there is no getter or "public" prop when checking the specs.
I checked Node's specs and what they do is that they declare Event and EventTarget in the same file, sharing the "kStop" symbol making it possible to access `event[kStop]` while still making it unreachable from the outside code:

```javascript
  stopImmediatePropagation() {
    // Spec mention "stopImmediatePropagation should set both "stop propagation"
    // and "stop immediate propagation" flags"
    // cf: from https://dom.spec.whatwg.org/#dom-event-stopimmediatepropagation
    this.stopPropagation();
    this[kStop] = true;
  }
```
then:
```javascript
    while (handler !== undefined &&
           (handler.passive || event?.[kStop] !== true)) {
```

But here, since EventTarget and Event are not in the same file nor in the same project I couldn't find a better way than this `event._stopImmediatePropagationFlag`.

Reply:
it's OK, and it's a sensible name. I don't know if by standards we have a way to know that thought ... if we do, maybe we should use the name whatever standards used (prefixed with `_` to do so).

Reply:
I didn't see anything in the spec for that (they are very vague), but I'll check again

Reply:
if there's nothing, I'm OK with this MR 👍

Reply:
Went through https://dom.spec.whatwg.org/#interface-event and I confirm I don't see any getter for "stop immediate propagation flag".